### PR TITLE
Move "search list" to "search"

### PR DESCRIPTION
**Pull request by rap1ds**
Built on top of #66 

Moves "search list" to "search". Changes the args parsing a bit.

### CODE REVIEW

File: src/sharetribe/flex_cli/commands/search_schema/set.cljs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                 (not (contains? types type))
+                 (conj (str "--type must be one of: " (str/join ", " types)))
+
+                 (not (contains? scopes scope))
+                 (conj (str "--scope must be one of: " (str/join ", " scopes)))
+
+                 (and default
+                      (= "boolean" type)
+                      (not (boolean? default)))
+                 (conj (str "--default must be boolean value when --type is boolean"))
```
Comment (ovan) [inline src/sharetribe/flex_cli/commands/search_schema/set.cljs]: What's a boolean value when passing string from commandline? "true" and "false"? If so, we could say "--default must be true or false when --type is boolean."

File: src/sharetribe/flex_cli/commands/search_schema/set.cljs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+                 (and default
+                      (= "long" type)
+                      (not (int? default)))
+                 (conj (str "--default must be integer value when --type is long")))]
```
Comment (ovan) [inline src/sharetribe/flex_cli/commands/search_schema/set.cljs]: "must be an integer..."

File: src/sharetribe/flex_cli/commands/search_schema/set.cljs [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                  body-params)
+
+         res (<? (do-post api-client "/search-schemas/set" query body))]
+
+     (println
+      (str
+       (if (= "public" (:scope params))
+         "Public data"
+         "Metadata")
+       " schema for " (:key params) " successfully set.")))))
```
Comment (ovan) [inline src/sharetribe/flex_cli/commands/search_schema/set.cljs]: No returning of strings from the command but direct println is the default way? I think I already forgot how our system here works.

Comment (rap1ds) [reply src/sharetribe/flex_cli/commands/search_schema/set.cljs]: Yeah, that's the default.

**Pull request merged**